src/components/SpecialResolution/SpecialResolutionForm.vue

```diff
@@ -1,11 +1,11 @@
 <template>
-  <v-card flat id="special-resolution-summary">
+  <v-card flat id="create-special-resolution-form">
```

Comment:
The pattern used in our UIs is to name this the same as the component name.

Also, this isn't a form (it's a card) so don't call it that.

Reply:
The ID looks OK but doesn't match the component filename. Would it make sense to name the component "CreateSpecialResolution" or "SpecialResolution" instead?

Reply:
we already have "SpecialResolution" for outer page, i will rename to CreateSpecialResolution

Reply:
Sounds good.